Sources/Sentry/SentryUIApplication.m

```diff
@@ -56,6 +56,105 @@ - (UIApplication *)sharedApplication
     return result;
 }
 
+- (NSArray<UIViewController *> *)relevantViewControllers
+{
+    NSMutableArray *result = [NSMutableArray array];
+
+    NSArray<UIWindow *> *windows = [self windows];
+    if ([windows count] == 0)
+        return nil;
+
+    for (UIWindow *window in windows) {
+        NSArray<UIViewController *> *vcs = [self relevantViewControllerFromWindow:window];
+        if (vcs != nil) {
+            [result addObjectsFromArray:vcs];
+        }
+    }
+
+    return result;
+}
+
+- (NSArray<UIViewController *> *)relevantViewControllerFromWindow:(UIWindow *)window
+{
+    UIViewController *rootViewController = window.rootViewController;
+    if (rootViewController == nil) {
+        return nil;
+    }
+
+    NSMutableArray<UIViewController *> *result = @[ rootViewController ].mutableCopy;
```

Comment:
I think this would be less allocations:
```suggestion
    NSMutableArray<UIViewController *> *result = [NSMutableArray<UIViewController *> arrayWithObject:rootViewController];
```